templates/components/common/facebook-like-button.html

```diff
@@ -0,0 +1,10 @@
+{{#if settings.facebook_like_button.enabled}}
```

Comment:
You should check for the existence of `settings.facebook_like_button` before using `enabled`.

Reply:
fixed.